src/components/tracked-controls.js

```diff
@@ -154,6 +156,7 @@ module.exports.Component = registerComponent('tracked-controls', {
     }
     previousButtonState.touched = buttonState.touched;
     this.el.emit('touch' + evtName, {id: id, state: previousButtonState});
+    // return true so handleButton knows something changed
```

Comment:
I thought I added as documentation of the return value in the jsdoc header for the function, did I mess it up?